playbooks/system-test.yaml

```diff
@@ -5,13 +5,18 @@
     toolbox_bin: '/usr/local/bin/toolbox'
 
   tasks:
-    - name: Build toolbox
+    - name: Set up build directory
       command: meson builddir
       args:
         chdir: '{{ zuul.project.src_dir }}'
+
+    - name: Build Toolbox
+      command: ninja -C builddir
+      args:
+        chdir: '{{ zuul.project.src_dir }}'
         creates: builddir/src/toolbox
```

Comment:
I am a bit surprised that this `creates` wasn't causing problems so far. Since `meson builddir` doesn't generate the binary, I would have expected it to complain.